### DISCUSSION TITLE
Upload to s3 - CSP error

### DISCUSSION THREAD

**Discussion by SantoshSrinivas79** [Budibase/budibase#6204] [Help] [answered]
I am trying to upload to AWS s3 in `us-east-1` on a self-hosted instance. Please advise

Getting the following errors in my console:

```
budibase-client.js?v=1.0.194:243 Refused to connect to 
**********************************
because it violates the following Content Security Policy directive: "connect-src 'self'
```

```
budibase-client.js?v=1.0.194:243 Refused to connect to 
***************************************
 because it violates the document's Content Security Policy.
```

<img width="701" alt="Screenshot 2022-06-07 at 9 39 42 AM" src="https://user-images.githubusercontent.com/1036163/172294710-41de3ee8-8330-4e10-8a5d-a733a847bd14.png">

**Comment by MoallaMed** [comment]
Same problem

**Reply by SantoshSrinivas79** [reply]
I also tried it with a bucket in `eu-west-1` and now I get this error: `net::ERR_CONNECTION_RESET`

<img width="873" alt="Screenshot 2022-06-07 at 6 30 06 PM" src="https://user-images.githubusercontent.com/1036163/172386350-57d49568-e193-4c73-909b-2e8f44706712.png">

**Comment by melohagan** [comment, marked as answer]
Hi @MoallaMed @SantoshSrinivas79 

There's a PR out at the moment that will hopefully resolve this once merged: https://github.com/Budibase/budibase/issues/6215

**Reply by SantoshSrinivas79** [reply]
@melohagan .. I used a bucket in the current CSP and getting the [above](https://github.com/Budibase/budibase/discussions/6204#discussioncomment-2897884) error. Can you please advise on this?

**Reply by SantoshSrinivas79** [reply]
This error was resolved due to a mistake .. the file name in the editor needs to be set correctly

**Reply by MoallaMed** [reply]
Hello @SantoshSrinivas79 can you tell me how you solved the problem I am still blocked ?
Thank you in advance

**Reply by SantoshSrinivas79** [reply]
i simply set up a new bucket at `eu-west-1` with right permissions and policies and then it worked just fine.

**Reply by MoallaMed** [reply]
![Sans titre](https://user-images.githubusercontent.com/102292139/173327080-a24fccaa-b714-4e14-af48-720329e411c7.png)
@SantoshSrinivas79  thank you for your reply , 
I did but a new problem appears, (the identifiers are correct) .

**Reply by MoallaMed** [reply]
@SantoshSrinivas79  Can you please tell me what are the exact steps to upload in s3, i still blocked .
